app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/ExpandedArticleFragment.java

```diff
@@ -0,0 +1,115 @@
+package edu.grinnell.grinnell_publications_android.Fragments;
+
+
+import android.graphics.drawable.Drawable;
+import android.support.annotation.Nullable;
+import android.os.Bundle;
+
+import android.support.design.widget.CollapsingToolbarLayout;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v4.app.Fragment;
+import android.support.v4.content.ContextCompat;
+
+import android.support.v7.widget.Toolbar;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import edu.grinnell.grinnell_publications_android.Models.Interfaces.UserInterface;
+import edu.grinnell.grinnell_publications_android.R;
+
+
+/**
+ * Fragment that displays the expanded article content.
+ * Fragment allows for favoriting of the article.
+ */
+
+public class ExpandedArticleFragment extends Fragment implements UserInterface {
+
+    private ImageView mHeaderImage;
```

Comment:
I like the consistency in naming the variables here! However please destroy these variables in the `onDestroyView(..) of the Fragment class and make sure you call the super method *after* you null all of these member variables out in the method. This will allow you to handle all the nulling out you need to do and the Fragment class will take care of the cleaning up via the super method :)